extra/modules/greenbids-real-time-data/src/test/java/org/prebid/server/hooks/modules/greenbids/real/time/data/v1/GreenbidsInferenceDataServiceTest.java

```diff
@@ -0,0 +1,132 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.v1;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.iab.openrtb.request.Banner;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Device;
+import com.iab.openrtb.request.Imp;
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.exception.GeoIp2Exception;
+import com.maxmind.geoip2.model.CountryResponse;
+import com.maxmind.geoip2.record.Country;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.data.GreenbidsInferenceDataService;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.data.ThrottlingMessage;
+import org.prebid.server.json.JacksonMapper;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.util.List;
+import java.util.function.UnaryOperator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class GreenbidsInferenceDataServiceTest {
+
+    @Mock
+    private DatabaseReader dbReader;
+
+    @Mock
+    private Country country;
+
+    private JacksonMapper jacksonMapper;
+
+    private TestBidRequestProvider testBidRequestProvider;
+
+    private GreenbidsInferenceDataService target;
+
+    @BeforeEach
+    public void setUp() {
+        final ObjectMapper mapper = new ObjectMapper();
```

Comment:
use the same provider as you use for the code being tested

Reply:
used `ObjectMapperProvider.mapper()`